packages/fast-check/src/check/arbitrary/definition/Value.ts

```diff
@@ -18,6 +18,7 @@ export class Value<T> {
    */
   readonly hasToBeCloned: boolean;
   /**
+   * @internal
```

Comment:
This one seems to be on an unrelated field? 🤔

Reply:
any private field or symbol on the public interface makes the types incompatible